src/app/zap-templates/common/ClustersHelper.js

```diff
@@ -166,6 +166,84 @@ function asPutCastType(zclType)
   }
 }
 
+function asChipCallback(atomic)
+{
+  switch (parseInt(atomic.atomicId)) {
+  case 0x00: // nodata / No data
```

Comment:
should we document where these cases come from (is it a spec chapter?).

Should we use constants?

Reply:
I have rewritten the method to not use the `atomicId` anymore.